bedrock/mozorg/templates/mozorg/home/includes/m24/ai-gallery.html

```diff
@@ -55,7 +55,7 @@ <h2 class="m24-c-intro-title">{{ ftl('m24-home-join-the-movement') }}</h2>
         desc=ftl('m24-home-donate-your-voice-to-make'),
         link_url='https://commonvoice.mozilla.org/'+ utm_params,
         cta_text=ftl('m24-home-donate-your-voice'),
-        link_attributes='data-cta-text="Donate your voice"'
```

Comment:
**Issue:** this gallery image is missing alt text still?

Reply:
That's intentional. It's a stock photo and doesn't supply any additional info.